content/security/advisories/rss.xml.haml

```diff
@@ -26,10 +26,12 @@
       %item
         %title
           = page.title
-        %date
+        %pubDate
```

Comment:
Why the change from "date" to "pubDate"?

Reply:
I see the reason in the commit comment.  I should have read the commit comment first.